benches/payout-stakers/src/main.rs

```diff
@@ -0,0 +1,88 @@
+use common::create_connection;
+use e2e::{
+    accounts::derive_user_account,
+    staking::{batch_bond, batch_nominate, bond, wait_for_era_completion},
+    staking::{
+        check_non_zero_payouts_for_era, validate, wait_for_full_era_completion, RewardDestination,
+    },
+    transfer::batch_endow_account_balances,
+};
```

Comment:
Consider moving this to common (since it is used both in the e2e and here). Alternatively lets create a task to do it later,

Reply:
I was thinking about the same, but let's leave it for later, I need to create a story for removing tech debt of e2e tests anyway, the good task for newcomers. :)

Reply:
Ok